include/dlaf/eigensolver/band_to_tridiag/api.h

```diff
@@ -7,18 +7,18 @@
 // Please, refer to the LICENSE file in the root directory.
 // SPDX-License-Identifier: BSD-3-Clause
 //
+
 #pragma once
 
 #include "dlaf/matrix/matrix.h"
 #include "dlaf/types.h"
 
-namespace dlaf {
-namespace eigensolver {
+namespace dlaf::eigensolver {
 
-template <class T, Device device>
+template <class T, Device D>
 struct ReturnTridiagType {
```

Comment:
Not your changes, but should this also be renamed `TridiagResult` like for the eigensolver result type?

Reply:
Yes, I can do it here definitely! @rasolca do you agree?